numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1738,12 +1737,19 @@ unpack_bits(PyObject *input, int axis)
     }
 
     /* Setup output shape */
-    for (i=0; i<PyArray_NDIM(new); i++) {
-        outdims[i] = PyArray_DIM(new, i);
-    }
+    memcpy(outdims, PyArray_DIMS(new), PyArray_NDIM(new) * sizeof(npy_intp));
```

Comment:
This bit really needs some review. Is there a reason `memcpy` wasn't used for this before? If not, did I use the correct `sizeof`?

Reply:
Is there any reason that memcpy _should_ be used here? It's less safe, and the loop should optimize to it anyway.